controllers/admin/AdminTranslationsController.php

```diff
@@ -2191,7 +2191,32 @@ public function getListModules($withInstance = false)
 
         return $modules;
     }
+    
+    /**
+     * @param $directory : name of directory
+     * @return array
+     */
+    protected function getSubjectMailContent($directory)
+    {
+        $subject_mail_content = array();
```

Comment:
```suggestion
        $subject_mail_content = [];
```

Reply:
I have added an array initialization based on your suggestion. But my opinion there was no need

Reply:
It's a requirement for passing PHPCSFixer check.